Explorer/Assets/Scripts/SceneRunner/Scene/AssetValidation.cs

```diff
@@ -0,0 +1,61 @@
+﻿using DCL.Diagnostics;
+using System;
+using UnityEngine;
+
+namespace SceneRunner.Scene
+{
+    public class AssetValidation
+    {
+        public const int AB_MIN_SUPPORTED_VERSION_WINDOWS = 15;
+        public const int AB_MIN_SUPPORTED_VERSION_MAC = 16;
+
+        public static bool ValidateSceneAssetBundleManifest(SceneAssetBundleManifest sceneAssetBundleManifest)
+        {
+            string errorText = "SceneID: " + sceneAssetBundleManifest.GetSceneID();
+            return ValidateVersion(sceneAssetBundleManifest.GetVersion(), errorText);
+        }
+
+        public static bool ValidateSceneAbDto_Hash(SceneAbDto sceneAbDto, string hash)
```

Comment:
I would change the paraphrasing here a little bit.

`hash` and `entityID` are actually the same, `entityID`. It either from the wearable, or from the scene. 

So, to keep naming consistent, and to avoid confusion, I would change the wording here to name it all `Entity ID`

You can even merge all methods into one, and add an extra parameter that determines which type on entity you are validating (Scene, or Wearable)